docs/miscellaneous.mdx

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 9
+---
+
+# Miscellaneous
+
+## Supported Video Codecs
+
+Fishjam currently supports the following video codecs:
+
+- **H.264**
+- **VP8**
+
+### Default Codec Behavior
```

Comment:
This is not true. VP8 is default in videoroom. If I would use sandbox with my own server implementation on sandbox - H.264 would be default.

Maybe this article should focus on explaining why room manager has VP8 (android emulator) and why h.264 should be used on production? 🤔